File: BitManipulation/GFG/Basic/OddEven/OddEven.cpp
// https://www.geeksforgeeks.org/problems/odd-or-even3618/1
#include<bits/stdc++.h>
using namespace std;
string oddEven(int n)
{
    // code here
    return (n & 1) ? "odd" : "even";
}
int main(void) {
    cout << oddEven(5);
    return 0;
}

File: BitManipulation/GFG/Basic/OddEven/OddEven.java
// https://www.geeksforgeeks.org/problems/odd-or-even3618/1
public class OddEven {
    static String oddEven(int n) {
        // code here
        return ((n & 1) == 0) ? "even" : "odd";
    }
    public static void main(String[] args) {
        System.out.println(oddEven(5));
    }
}
